docs/ПереопределениеМетодовБСП.md

```
@@ -50,15 +50,15 @@
   
   Если модуль-аналог новый добавить функцию в области "СлужебныеПроцедурыИФункции", где пбп_ИмяМодуля для серверных модулей "пбп_ОбщегоНазначенияПовтИсп.", для клиентских "пбп_ОбщегоНазначенияКлиент"
   
-  ```1C (BSL)
+  ```BSL
   Функция ПолучитьМодуль()
       Возврат пбп_ИмяМодуля.ОбщийМодуль(ИмяМодуляБСП, ИмяМодуляАналогаБСП);
   КонецФункции
   ```
   ### 2.2 Создание методов в модуле-аналоге
```

Markdownlint

Headings must start at the beginning of the line

(MD023, heading-start-left)